config/clusters/2i2c-uk/support.values.yaml

```diff
@@ -0,0 +1,21 @@
+prometheusIngressAuthSecret:
+  enabled: true
+
+prometheus:
+  server:
+    ingress:
+      enabled: true
+      hosts:
+        - prometheus.community.uk.2i2c.cloud
```

Comment:
Let's get rid of the `.community` prefix, and just use `.uk.2i2c.cloud` both here and for the staging hub?

Reply:
Done